packages/paste-website/src/pages/utilities/aspect-ratio/index.mdx

```diff
@@ -0,0 +1,151 @@
+---
+title: Aspect Ratio - Utilities
+package: '@twilio-paste/aspect-ratio'
+description: The Aspect Ratio utility is used to embed images, video, media, components, or other HTML elements. It prevents skewing, cropping, etc.
+---
+
+import {graphql} from 'gatsby';
+import {Link} from 'gatsby';
+import Img from 'gatsby-image';
+import {Anchor} from '@twilio-paste/anchor';
+import {Absolute} from '@twilio-paste/absolute';
+import {AspectRatio} from '@twilio-paste/aspect-ratio';
+import {Box} from '@twilio-paste/box';
+import Changelog from '@twilio-paste/aspect-ratio/CHANGELOG.md';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+import {SidebarCategoryRoutes} from '../../../constants';
+
+export const pageQuery = graphql`
+  {
+    allPasteUtility(filter: {name: {eq: "@twilio-paste/aspect-ratio"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    file(sourceInstanceName: {eq: "assets"}, relativePath: {eq: "images/anchor-dont-1@4x.png"}) {
+      childImageSharp {
+        fluid(maxWidth: 500) {
+          ...GatsbyImageSharpFluid
+        }
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Aspect Ratio"
+  categoryRoute={SidebarCategoryRoutes.UTILITIES}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/utilities/aspect-ratio"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/utilities-aspect-ratio--default"
+  data={props.data.allPasteUtility.edges}
+/>
+
+***
+
+<content>
+
+## Guidelines
+
+### About Aspect Ratio
+
+The Aspect Ratio component dynamically modifies the height of an element based on its
+current width to match a height-to-width ratio. For example, a 4:3 ratio would make
+an element with a width of 400px have a height of 300px. This updates in real time
+when you resize the screen.
+
+### Accessibility Information
+
+Be considerate of smaller devices when using this component. If necessary, you can
+modify the aspect ratio property at certain media query breakpoints.
```

Comment:
> modify the aspect ratio property at certain media query breakpoints

How?

Reply:
Someone could write a function that changes the prop value a certain breakpoints. Do I need to mention that here? Or leave it open ended?

Reply:
So this is a thing where I mentioned to Rich we should experiment to see if we can use the styled-system thing of `ratio={['4:3', '5:3']}`

Alternatively, the consumer's wrapping code can just pass in a different prop based on window size, if they have some kind of window resize listener.

Reply:
> So this is a thing where I mentioned to Rich we should experiment to see if we can use the styled-system thing of ratio={['4:3', '5:3']}

❤️ 

I love that I drop a question and you both come to where I was aiming for.

I would LOVE to get the styled-system type arrays to work for this.

Reply:
Go team!
I'll create a ticket in the backlog for that investigation.

Reply:
Cool, I agree with that. My only other concern is that considering small screens and using breakpoints really has nothing to do with accessibility. Can we change this heading?

Reply:
Yeah. That heading doesn't really make sense anymore. What do you think about just dropping the heading, and the text drops down to a second paragraph under Composition Notes?

Reply:
> text drops down to a second paragraph under Composition Notes?

I buy it